examples/acceptance_tests/048/lib/tests.ak

```diff
@@ -45,3 +45,25 @@ test foo_4() {
       False
   }
 }
+
+type Seasons {
+  Winter
+  Spring
+  Summer
+  Fall
+}
+
+fn is_cold(season, hour) {
+  when season is {
+    Winter | Fall ->
+      True
+    _ if hour >= 18 ->
+      True
+    _ ->
+      False
+  }
+}
+
+test foo_5() {
+  !is_cold(Spring, 15) && !is_cold(Summer, 22)
```

Comment:
`!is_cold(Summer, 22)` returns False? is that intentional? or do we want to remove the `!`

Reply:
Right, extra `!` here. The code doesn't even reach that branch anyway at the moment.

Reply:
I had a feeling